configs/containers/specs/jedi-ci.yaml

```diff
@@ -1,18 +1,66 @@
-  ### spack-stack-1.6.0 / skylab-7.x.y containers for fv3-jedi and mpas-jedi (but not for ufs-jedi)
-  specs: [base-env@1.0.0, jedi-base-env@1.0.0, ewok-env@1.0.0, jedi-fv3-env@1.0.0,
-    jedi-mpas-env@1.0.0, bacio@2.4.1, bison@3.8.2, bufr@12.0.1, ecbuild@3.7.2, eccodes@2.33.0, ecflow@5,
-    eckit@1.24.5, ecmwf-atlas@0.38.1 +fckit +trans +tesselation +fftw, fiat@1.2.0, ectrans@1.2.0 +fftw,
-    eigen@3.4.0, fckit@0.11.0, fms@2024.02, g2@3.5.1, g2tmpl@1.13.0,
-    gsibec@1.2.1, hdf@4.2.15, hdf5@1.14.3, ip@5.0.0, jasper@2.0.32, jedi-cmake@1.4.0,
-    libpng@1.6.37, nccmp@1.9.0.1, netcdf-c@4.9.2, netcdf-cxx4@4.3.1, netcdf-fortran@4.6.1,
-    parallelio@2.6.2, parallel-netcdf@1.12.3, py-eccodes@1.5.0, py-f90nml@1.4.3,
-    py-gitpython@3.1.40, py-h5py@3.8.0, py-numpy@1.22.3,
-    py-pandas@1.5.3, py-pip, py-pyyaml@6.0, py-scipy@1.11.4, py-shapely@1.8.0, py-xarray@2023.7.0,
-    sp@2.5.0, udunits@2.2.28, w3emc@2.10.0, nco@5.1.6, esmf@8.6.1, mapl@2.46.3,
-    zlib-ng@2.1.5, zstd@1.5.2, odc@1.5.2, shumlib@macos_clang_linux_intel_port,
-    awscli-v2@2.13.22, py-globus-cli@3.16.0,
+# spack-stack containers are configured for fv3-jedi and mpas-jedi, but not for ufs-jedi.
+  specs: [
+    base-env@1.0.0,
+    jedi-base-env@1.0.0,
+    ewok-env@1.0.0,
+    jedi-fv3-env@1.0.0,
+    jedi-mpas-env@1.0.0,
+    bacio@2.4.1,
+    bison@3.8.2,
+    bufr@12.1.0,
+    ecbuild@3.7.2,
+    eccodes@2.33.0,
+    ecflow@5.11.4,
+    eckit@1.24.5,
+    ecmwf-atlas@0.38.1 +fckit +trans +tesselation +fftw,
+    fiat@1.2.0,
+    ectrans@1.2.0 +fftw,
+    eigen@3.4.0,
+    fckit@0.11.0,
+    fms@2023.04,
+    g2@3.5.1,
+    g2tmpl@1.13.0,
+    gsibec@1.2.1,
+    hdf@4.2.15,
+    hdf5@1.14.3,
+    ip@5.0.0,
+    jasper@2.0.32,
+    jedi-cmake@1.4.0,
+    libpng@1.6.37,
+    nccmp@1.9.0.1,
+    netcdf-c@4.9.2,
+    netcdf-cxx4@4.3.1,
+    netcdf-fortran@4.6.1,
+    parallelio@2.6.2,
+    parallel-netcdf@1.12.3,
+    py-eccodes@1.5.0,
+    py-f90nml@1.4.3,
+    py-gitpython@3.1.40,
+    py-h5py@3.8.0,
+    py-numpy@1.25,
+    py-pandas@1.5.3,
+    py-pip,
+    py-pyyaml@6.0,
+    py-scipy@1.11.4,
+    py-shapely@1.8.0,
```

Comment:
not yet, but once shapely gets updated you'll want to look at all the other py-* packages, too

Reply:
agreed, I'll follow up with that.